runestone/unittest_base.py

```diff
@@ -41,7 +40,7 @@ def setUpModule(self):
         if self.exit_status_success:
             self.assertFalse(p.returncode)
         # Run the server. Simply calling ``runestone serve`` fails, since the process killed isn't the actual server, but probably a setuptools-created launcher.
-        self.runestone_server = subprocess.Popen(['python', '-m', 'runestone', 'serve', '--port', PORT])
+        self.runestone_server = subprocess.Popen([sys.executable, '-m', 'runestone', 'serve', '--port', PORT])
```

Comment:
I feel like we are going back and forth here in that sys.executable doesn't work when the system thinks executable is runestone.  I thought I had to change this back to python in this exact same place.

Reply:
I don't see it -- the original commit of 63bf26e4cbd3dd771e69544e35bde94678979f03 was `self.runestone_server = subprocess.Popen(['python', '-m', 'runestone', 'serve', '--port', PORT])`. I do remember this on the server, where NGINX's emperor mode broke `sys.executable` when trying to do a runestone build for a problem written by the instructor in the admin interface.

Reply:
Yes, that was the case I was thinking of.  So in my mind that makes sys.executable unreliable.

Reply:
I see you merged this, so I'm assuming you're OK with this.

For the record, from https://docs.python.org/3/library/sys.html#sys.executable is "A string giving the absolute path of the executable binary for the Python interpreter, on systems where this makes sense. If Python is unable to retrieve the real path to its executable, sys.executable will be an empty string or None."

In the case of testing, we can assume CPython, hence a correct `sys.executable`.